seleniumbuilder/content/html/builder/record.js

```diff
@@ -81,25 +81,25 @@ builder.record.stop = function() {
 
 builder.record.continueRecording = function(insertionIndex) {
   jQuery('#record-panel').show();
-  
+
```

Comment:
If you could fix it (along with the commented-out line below), I would be very grateful. It's very hard to see what the diff does like this.